services/ext/api.go

```diff
@@ -1066,8 +1066,12 @@ func (api *PublicAPI) UpsertSavedAddress(ctx context.Context, sa wallet.SavedAdd
 	return api.service.messenger.UpsertSavedAddress(ctx, sa)
 }
 
-func (api *PublicAPI) DeleteSavedAddress(ctx context.Context, address ethcommon.Address, ens string, isTest bool) error {
-	return api.service.messenger.DeleteSavedAddress(ctx, address, ens, isTest)
+func (api *PublicAPI) DeleteSavedAddress(ctx context.Context, address ethcommon.Address, isTest bool) error {
```

Comment:
This is to have access to waku :(

Reply:
> It's strange to see the saved address APIs here. However, I assume this is due to our current architecture.

Yes, it is, I agree, but having Add/Delete saved address endpoints directly exposed as a part of wallet services avoids sending sync messages when the client uses them and that's why I removed them from there and kept them only here, to avoid misleading usage.

Logically those calls belong to the wallet service, which means once we establish an architecture we should move them there, but till then we should have them in a single place.